### PR TITLE
Improve examples&documentation

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: examples/runtime/runtime.go [hunk 1]
```diff
@@ -1,55 +1,81 @@
...
+// _deploy is called after contract deployment or update, it'll be called
+// in deployment transaction and if call update method of this contract.
 func _deploy(_ interface{}, isUpdate bool) {
 	if isUpdate {
-		Log("_deploy method called before contract update")
+		Log("_deploy method called after contract update")
 		return
 	}
-	Log("_deploy method called before contract creation")
+	Log("_deploy method called aftet contract creation")
```
Comment (AnnaShaleva) [inline examples/runtime/runtime.go:30]: Typo.

**Review by AnnaShaleva — COMMENTED** [review]
I faced with a problem while checking some of our examples: our privnet image fails to built with the following error:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ make env_image
=> Building env image
Sending build context to Docker daemon  124.4MB
Step 1/18 : FROM golang:1.18-alpine as builder
 ---> a7fc04766ca8
Step 2/18 : RUN go version
 ---> Using cache
 ---> e30d977fb8e6
Step 3/18 : RUN set -x     && apk add --no-cache git make     && mkdir -p /tmp
 ---> Using cache
 ---> ac037564b737
Step 4/18 : COPY . /neo-go
 ---> Using cache
 ---> b8d663d856bd
Step 5/18 : WORKDIR /neo-go
 ---> Using cache
 ---> 37fd113d416a
Step 6/18 : ARG REPO=repository
 ---> Using cache
 ---> 884428c64f57
Step 7/18 : ARG VERSION=dev
 ---> Using cache
 ---> ba9310abaa64
Step 8/18 : RUN make build
 ---> Running in 21b948478f30
make: /bin/sh: Operation not permitted
make: *** [Makefile:80: deps] Error 127
The command '/bin/sh -c make build' returned a non-zero code: 2
Makefile:109: recipe for target 'env_image' failed
make: *** [env_image] Error 2
```

This error connected with an old Docker engine version:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ docker --version
Docker version 19.03.8, build afacb8b7f0
```

The problem is that we use alpine base image to build neo-go image, and alpine require at least 20.10.0 Docker engine version: https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2.

The problem is fixed by updating Docker image according to the documentation: https://docs.docker.com/engine/install/ubuntu/.

I think we should keep in mind this issue while preparing to the neo-go workshop. Maybe it's worth to mention somewhere in workshop code.

**Comment by roman-khimov** [comment]
I've recently upgraded my OS, so never noticed that. Should be mentioned somewhere though if we know it now.

**Review by fyrchik — APPROVED** [review]

File: examples/runtime/runtime.go (hunk 1 above)
Comment (roman-khimov) [reply examples/runtime/runtime.go]: Fixed

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**